cdc/sink/mq_test.go

```diff
@@ -182,3 +190,30 @@ func (s mqSinkSuite) TestKafkaSinkFilter(c *check.C) {
 		c.Assert(errors.Cause(err), check.Equals, context.Canceled)
 	}
 }
+
+func (s mqSinkSuite) TestPulsarSinkEncoderConfig(c *check.C) {
+	defer testleak.AfterTest(c)()
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	err := failpoint.Enable("github.com/pingcap/ticdc/cdc/sink/producer/pulsar/MockPulsar", "return(true)")
+	c.Assert(err, check.IsNil)
+
+	uri := "pulsar://127.0.0.1:1234/kafka-test?max-batch-size=1" +
+		"&max-message-bytes=4194304&max-batch-size=1048576"
```

Comment:
Why there are two `max-batch-size`

Reply:
It was in the original Kafka test case. Fixed.